libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -852,11 +852,7 @@ - (void)setupWithCredentials:(SFOAuthCredentials*) credentials {
 
 - (void)restartAuthentication {
     @synchronized (self.authBlockList) {
-        if (!self.authenticating) {
-            [self log:SFLogLevelWarning format:@"%@: Authentication manager is not currently authenticating.  No action taken.", NSStringFromSelector(_cmd)];
-            return;
-        }
-        [self log:SFLogLevelInfo format:@"%@: Restarting in-progress authentication process.", NSStringFromSelector(_cmd)];
+        [self log:SFLogLevelInfo format:@"%@: Restarting authentication process.", NSStringFromSelector(_cmd)];
```

Comment:
I went through the flow, and the use case I missed is that we're still showing the initial alert about Safari opening to authenticate.  Shouldn't we just remove that with the embedded view?  I don't see that it brings value when we switch to an inline authentication process.

The process I was commenting on was from the "Safari VC close" use case, in which case the authentication is still technically in progress, and should be fine.